src/app/external-cluster-wizard/steps/external-cluster/provider/aks/component.ts

```diff
@@ -99,7 +100,8 @@ export class AKSClusterSettingsComponent
   isLoadingVmSizes: boolean;
   isLoadingNodePoolVersions: boolean;
   vmSizes: string[] = [];
-  nodePoolVersions: string[] = [];
+  nodePoolVersionsForMD: string[] = [];
+  kubernetesVersionsForClusterCreation: string[] = [];
```

Comment:
- IMO, we should change to. `kubernetesVersions` we know we are at create wizard so it is for creation
- Keep consistent with other EKS/GKE -> we use `kubernetesVersions`